plugins/secretstores/README.md

```diff
@@ -0,0 +1,6 @@
+# Secret-Stores
+
+This folder contains the plugins for the secret-store functionality:
+
+* jose: Javascript Object Signing and Encryption
+* os: Native tooling provided on Linux, MacOS, or Windows.
```

Comment:
Should this be part of this PR? Ok for me, just asking...

Reply:
possibly not :) but when I went to go add this to the docs I occurred to me that there should be another doc to make finding things a bit easier.